.devcontainer/bazel-base/Dockerfile

```diff
@@ -0,0 +1,130 @@
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+#  MAGMA_ROOT is needed by python tests (e.g. freedomfi_one_tests in enodebd)
+ENV MAGMA_ROOT=/workspaces/magma
+
+RUN echo "Install general purpose packages" && \
+    apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y --no-install-recommends \
+        apt-transport-https \
+        apt-utils \
+        # dependencies of FreeDiameter
+        bison \
+        build-essential \
+        ca-certificates \
+        cmake \
+        curl \
+        gcc \
+        git \
+        gnupg2 \
+        g++ \
+        # dependency of mobilityd (tests)
+        iproute2 \
+        # dependency of python services (e.g. magmad)
+        iputils-ping \
+        flex \
+        libconfig-dev \
+        # dependency of @sentry_native//:sentry
+        libcurl4-openssl-dev \
+        # dependencies of oai/mme
+        libczmq-dev \
+        libgcrypt-dev \
+        libgmp3-dev \
+        libidn11-dev \
+        libsctp1 \
+        # dependency of sctpd
+        libsctp-dev \
+        libssl-dev \
+        # dependency of pip systemd
+        libsystemd-dev \
+        lld \
+        # dependency of python services (e.g. magmad)
+        net-tools \
+        # dependency of python services (e.g. pipelined)
+        netbase \
+        python${PYTHON_VERSION} \
+        python-is-python3 \
+        software-properties-common \
+        # needed for post install script of libfolly-dev
+        sudo \
+        # dependency of python services (e.g. magmad)
+        systemd \
+        unzip \
+        # dependency of liagent
+        uuid-dev \
+        vim \
+        wget \
+        zip
+
+RUN apt-get install -y --no-install-recommends \
+        libtool=2.4.6-14 && \
+    echo "Install Folly dependencies" && \
+    apt-get install -y --no-install-recommends \
+        libgoogle-glog-dev \
+        libgflags-dev \
+        libboost-all-dev \
+        libevent-dev \
+        libdouble-conversion-dev \
+        libboost-chrono-dev \
+        libiberty-dev && \
+    echo "Install libtins and connection tracker dependencies" && \
+    apt-get install -y --no-install-recommends \
+        libpcap-dev=1.9.1-3 \
+        libmnl-dev=1.0.4-2
+
+## Install Fmt (Folly Dep)
+RUN git clone https://github.com/fmtlib/fmt.git && cd fmt && \
+    mkdir _build && cd _build && \
+    cmake .. -DFMT_TEST=0 && \
+    make -j"$(nproc)" && \
+    make install && \
+    cd / && \
+    rm -rf fmt
+
+# Facebook Folly C++ lib
+# Note: "Because folly does not provide any ABI compatibility guarantees from
+#        commit to commit, we generally recommend building folly as a static library."
+# Here we checkout the hash for v2021.02.22.00 (arbitrary recent version)
+RUN git clone https://github.com/facebook/folly && cd folly && \
+    git checkout tags/v2021.02.15.00 && \
+    mkdir _build && cd _build && \
+    cmake .. && \
+    make -j"$(nproc)" && \
+    make install && \
+    cd / && \
```

Comment:
Comparing to line 97, where a relativ change of directory was made, I would also make a `cd ..` here. Or use absolute path in line 97 as well, e.g. `cd /folly`.